include/clover_field_order.h

```diff
@@ -734,34 +734,34 @@ namespace quda {
 	void save() {
 	  if (backup_h) errorQuda("Already allocated host backup");
 	  backup_h = safe_malloc(bytes);
-	  cudaMemcpy(backup_h, clover, bytes, cudaMemcpyDeviceToHost);
-	  if (norm_bytes) {
-	    backup_norm_h = safe_malloc(norm_bytes);
-	    cudaMemcpy(backup_norm_h, norm, norm_bytes, cudaMemcpyDeviceToHost);
-	  }
-	  checkCudaError();
-	}
+          qudaMemcpy(backup_h, clover, bytes, qudaMemcpyDeviceToHost);
```

Comment:
This should be a no-tune variant since the `save` and `load` methods are called exclusively by `preTune` and `postTune`, respectively.

Reply:
c1df95f00